tasks/pkg-debian.yml

```diff
@@ -117,3 +117,11 @@
 
 - include_tasks: pkg-debian/install-latest.yml
   when: datadog_agent_debian_version is not defined
+
+- name: Install latest datadog-signing-keys package
+  apt:
+    name: datadog-signing-keys
+    state: latest  # noqa 403
```

Comment:
The dependency we add to the Agent should not clash with this (eg: we should require a version "greater or equal" than X in the Agent).

Reply:
Yeah. "latest" will always be the desired state for this package. The agent dependency will only limit the lower bound of the version, which is compatible with "latest".